website/content/docs/auth/jwt/oidc-providers/adfs.mdx

```diff
@@ -0,0 +1,344 @@
+   $ vault secrets enable -path=<ADFS_KV_PLUGIN_PATH> kv-v2
+   ```
+   For example:
+
+  <CodeBlockConfig hideClipboard>
+
+  ```shell-session
+   $ vault secrets enable -path=adfs-kv kv-v2
+  ```
+
+  </CodeBlockConfig>
+
+1. Create a read-only policy against the KV plugin for ADFS:
+   ```shell-session
+   $ vault policy write <RO_ADFS_POLICY_NAME> - << EOF
+   # Read and list policy for the ADFS KV mount
+   path "<ADFS_KV_PLUGIN_PATH>/*" {
+     capabilities = ["read", "list"]
+   }
+   EOF
+   ```
+   For example:
+
+  <CodeBlockConfig hideClipboard>
+
+  ```shell-session
+   $ vault policy write read-adfs-test - << EOF
+   # Read and list policy for the ADFS KV mount
+   path "adfs-kv/*" {
+     capabilities = ["read", "list"]
+   }
+   EOF
+  ```
+
+  </CodeBlockConfig>
+
+1. Write a test value to the KV plugin:
+  ```shell-session
+  $ vault kv put <ADFS_KV_PLUGIN_PATH>/test test_key="test value"
+   ```
+   For example:
+
+  <CodeBlockConfig hideClipboard>
+
+  ```shell-session
+  $ vault kv put adfs-kv/test test_key="test value"
+  ```
+
+  </CodeBlockConfig>
+
+Now you can create a Vault group and link to an AD group:
+
+<Tabs>
+
+<Tab heading="Vault CLI">
+
+1. Create an external group in Vault and save the group ID to a file named
+   `group_id.txt`:
+  ```shell-session
+  $ vault write \
+    -format=json \
+    identity/group name="<YOUR_NEW_VAULT_GROUP_NAME>" \
+    policies="<RO_ADFS_POLICY_NAME>" \
+    type="external" | jq -r ".data.id" > group_id.txt
+  ```
+1. Retrieve the mount accessor for the ADFS authentication method and save it to
+   a file named `accessor_adfs.txt`:
+   ```shell-session
+   $ vault auth list -format=json | \
+     jq -r '.["<YOUR_OIDC_MOUNT_PATH>/"].accessor' > \
+     accessor_adfs.txt
+   ```
+1. Create a group alias:
+   ```shell-session
+   $ vault write identity/group-alias \
+     name="<YOUR_EXISTING_AD_GROUP>"  \
+     mount_accessor=$(cat accessor_adfs.txt) \
+     canonical_id="$(cat group_id.txt)"
+   ```
+1. Login to Vault as an AD user who is a member of YOUR_EXISTING_AD_GROUP.
+1. Read your test value from the KV plugin:
+  ```shell-session
+  $ vault kv list -mount=secret <ADFS_KV_PLUGIN_PATH>/test
+  ```
+
+</Tab>
+
+<Tab heading="Vault UI">
+
+1. Open the Vault UI.
+1. Select **Access**. 
+1. Select **Groups**.
+1. Click **Create group**.
+1. Follow the prompts to create an external group with the following
+   information:
+     - Name: your new Vault group name
+     - Type: `external`
+     - Policies: the read-only ADFS policy you created. For example,
+       `read-adfs-test`.
+1. Click on **Add alias** and follow the prompts to map the Vault group name
+   to an existing group on your AD:
+   - Name: the name of an existing AD group (**must match exactly**).
+   - Auth Backend: `<YOUR_OIDC_MOUNT_PATH>/ (oidc)`
+1. Login to Vault as an AD user who is a member of the aliased AD group.
+1. Read your test value from the KV plugin.
+
+</Tab>
+
+</Tabs>
+
+## Next steps
+
+TBD
```

Comment:
Is this still work in progress?  Or, if there is not specific next step, you should remove the section.

```suggestion
```

Reply:
Hello, it's finish
You can remove Next steps
Thank you